### DISCUSSION TITLE
Select Option - reset field B after selecting field A

### DISCUSSION THREAD

**Discussion by tanetrail** [react-hook-form/react-hook-form#4171] [Q&A]
form has 2 select fields, FieldA of type Select  and FieldB of type Select. FieldB selectoption values are dependent on FieldA value. i could able to populate the selectoptions for FieldB using watch on FieldA but How to clear or set a default value in FieldB after selecting a value in FieldA

**Comment by nbnsresta** [comment]
Try this.
```
const valueA = watch("fieldA")

React.useEffect(() => {
  setValue("fieldB", "");
}. [valueA, setValue]);
```
